python/Ganga/new_tests/TestProxy.py

```diff
@@ -79,7 +85,7 @@ def test_dict_attributes(self):
         self.assertFalse(hasattr(proxied, '_impl'))
 
         # A proxy instance should only have _impl in its dict
-        self.assertEqual(self.p.__dict__.keys(), ['_impl'])
+        self.assertEqual(self.p.__dict__.keys(), ['_impl', '_proxyObject'])
```

Comment:
The object being tested DOES NOT have a _proxyObject hence the assetTrue test would fail. After the test failing when I test the same object I expect not to have a _proxyObject (still) the assetEqual fails because it now has one. I don't know where it gets it from but it does during the test.

At least when I run it. This is a concerning bug and I don't know what could lead to this, but frankly as I reverted the change I'm letting this one fall by the wayside unless anyone else wants to do any digging.